python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -291,12 +291,12 @@ def index_load(self, this_id):
                     obj = self._make_empty_object_(this_id, cat, cls)
                 except Exception as err:
                     raise IOError('Failed to Parse information in Index file: %s. Err: %s' % (fn, err))
-            this_cache = obj._index_cache
+            this_cache = obj._index_cache_dict
```

Comment:
I rally don't like circumventing the `_index_cache` getter here. If there are problems with it then we should look at addressing those. I'm not a big fan of the getter anyway as it's doing more blurring of the line between objects and the registry.